composer.json

```diff
@@ -30,7 +32,7 @@
         "friendsofphp/php-cs-fixer": "^2.8",
         "friendsoftwig/twigcs": "^3.1.2",
         "symfony/http-client": "^4.3|^5.0",
-        "symfony/phpunit-bridge": "^4.3|^5.0",
+        "symfony/phpunit-bridge": "^5.1@dev",
```

Comment:
Why the change in version constraints? We may want to leave this at `^4.3|^5.0`. `minimum-stability: dev` will use the latest dev releases.